### PR TITLE
Update docker for Ubuntu 22.04

### PR DESCRIPTION
**Pull request by bhufmann**
This PR consists of 2 commits:

- Move docker/ubuntu-18.04 to root docker directory
- Update Dockerfile and README.md for Ubuntu 22.04

Signed-off-by: Bernd Hufmann <bernd.hufmann@ericsson.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,15 +59,15 @@ Other commit information:
 
 * [How to format the message][commit-message-message]
 
-## How to build the docker image (tag 18.04)
+## How to build the docker image (tag 22.04)
 
 ```bash
-docker build --no-cache --build-arg version=18.04 --build-arg strip=true  -t eclipse/tracecompass-build-env:18.04 .
+docker build --network host --no-cache --build-arg version=22.04 --build-arg strip=true  -t eclipse/tracecompass-build-env:22.04 .
```
Comment (marcdumais-work) [inline CONTRIBUTING.md:65]: > --build-arg version=22.04 --build-arg strip=true

It looks like the image was initially designed to take the ubuntu version as a `build argument`, as well as whether to "strips" it or not. I do not see these arguments used in the image any more, so they could be removed to simplify the build command. In any case, I would think in most cases, more than the Ubuntu version will need be updated, for the image to work with a new Ubuntu, so it's ok not having this configuration.

Comment (marcdumais-work) [inline CONTRIBUTING.md:65]: > --network host 

Did you have a specific reason to add this? This seems to be a new option for "docker build". Di you need it when building locally?

Comment (bhufmann) [reply CONTRIBUTING.md:65]: I need that for some reason on my laptop.

Comment (marcdumais-work) [reply CONTRIBUTING.md:65]: I think it's ok - it should not affect the resulting image, so we can do whatever is needed to successfully build.

**Review by marcdumais-work — APPROVED** [review]
(symbolic approval) LGTM - just a minor comment to remove unused build arguments and we're good to build, push and test the image!

File: CONTRIBUTING.md (hunk 1 above)
Comment (marcdumais-work) [reply CONTRIBUTING.md]: Latest version looks good!

**Pull request merged**